### PR TITLE
Add sub actions

### PR DESCRIPTION
**Pull request by arturogonzalez58**
Prior to this PR we only have the main workflow to intercept a service. Now the user has the ability to split the workflow in different steps. This PR adds the intercept, connect, and login action. Each of these actions has a post handler to execute a clean-up action after the action is concluded. 

Also, the node_modules directory is added as the Github Action requires, [see](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github).

The actions were tested using the [development repository](https://github.com/datawire/development-telepresence-intercept) and [this](https://github.com/arturogonzalez58/emojivoto/) emojivoto repository.

### CODE REVIEW

File: intercept/action.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: 'Telepresence Intercept'
+description: 'Intercept a service'
+inputs:
+  service_name:
+    description: The name of the ervice to intercept
```
Comment (hojenkai) [inline intercept/action.yaml:5]:
```suggestion
    description: The name of the service to intercept
```

**Comment by hojenkai** [comment]
Do we have a `package.json` and `package-lock.json` files that we should include?

**Comment by arturogonzalez58** [comment]
> Do we have a `package.json` and `package-lock.json` files that we should include?

Thanks, I forgot it

**Review by KeoniMuller — APPROVED** [review]

File: login/logout.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const core = require('@actions/core')
+const exec = require('@actions/exec');
+
+const telepresenceLogout = async function(){
+    try {
+        await exec.exec('telepresence', ['logout']);
```
Comment (hojenkai) [inline login/logout.js:6]: now that TP has released a new version I've been doing some tests with the emoji service and looks like when using v2.6.0 in the CI (which for now doesn't work) and then using v2.5.8 in another version the cluster (or traffic agent) expects a client at version 2.6.0 due to the first run. My question is: Should we uninstall TP after we run integration tests so we left the cluster as it originally was?

Comment (hojenkai) [reply login/logout.js:6]: please disregard - as we spoke in our sync up. This is due to the side effects of installing TP using the `install` command

**Pull request merged**